doc/user_manual/model.tex

```diff
@@ -1561,6 +1561,21 @@ \subsection{EnsembleModel}
         otherwise an error will be raised.
   \end{itemize}
 \end{itemize}
+
+\nb \textcolor{red} { \textbf{ It is crucial to understand that the choice of the \xmlNode{DataObject} used as
+ \newline \xmlNode{TargetEvaluation} determines how the data are going to be transferred from a model to
+  the other. If for example the chain of models is $A \rightarrow B$:}}
+\begin{itemize}
+  \item \textcolor{red} { \textbf{ If model $B$ expects as Input scalars and outputs time-series, the \xmlNode{TargetEvaluation}  
+  of the  model $B$ will be a \textit{HistorySet} and the  \xmlNode{TargetEvaluation} of the model $A$ will be either 
+  a \textit{PointSet} or a \textit{DataSet} (where the output variables that need to be transferred to the model $A$ are scalars) }    }
+   \item \textcolor{red} { \textbf{ If model $B$ expects as Input scalars and time-series and outputs time-series or scalars or both, the \xmlNode{TargetEvaluation}  
+  of the  model $B$ will be a \textit{DataSet} and the \newline  \xmlNode{TargetEvaluation} of the model $A$ will be either 
+  a \textit{HistorySet} or a \textit{DataSet}  }    }
+  \item \textcolor{red} { \textbf{ If both model $A$ and $B$ expect as Input scalars and output scalars, the \xmlNode{TargetEvaluation}  
+  of the  both models  $A$  and $B$ will be  \textit{PointSet}s  }  }
+\end{itemize}
+
```

Comment:
I wonder, could we create a simple table to help summarize the different data object features? Something like
```tex
\begin{table}
  \centering
  \begin{tabular}{c|c|c}
    Data Object & Input Space & Output Space \\
    DataSet     & any       & any \\
    PointSet    & scalars     & scalars \\
    HistorySet  & scalars     & vectors
  \end{tabular}
\end{table}
```
I'm not sure where in the document would be the best place for it, or even if it would help at all.

Reply:
added